test/native/setup.js

```diff
@@ -53,6 +53,7 @@ jest.mock( '@wordpress/react-native-bridge', () => {
 		subscribeUpdateCapabilities: jest.fn(),
 		subscribeShowNotice: jest.fn(),
 		subscribeParentGetHtml: jest.fn(),
+		subscribeShowEditorHelp: jest.fn(),
```

Comment:
💯 Good idea to add this here. I overlooked this in my own implementations.

Reply:
Thanks!